### ISSUE TITLE
Crash when click 'use default layout' in menu.

### ISSUE THREAD

**Issue by gzhechu** [spyder-ide/spyder#8318]
### What steps will reproduce the problem?

Crash when click 'use default layout' in menu, and then popup this dialog to report problem.

### Traceback
```python-traceback
TypeError: _start_server() missing 1 required positional argument: 'port'
```

## Versions

* Spyder version: 3.3.1 
* Python version: 3.7.0
* Qt version: 5.9.6
* PyQt5 version: 5.9.2
* Operating System: Linux 4.15.0-39-generic

### Dependencies

```
pyflakes >=0.6.0  :  2.0.0 (OK)
pycodestyle >=2.3 :  2.4.0 (OK)
pygments >=2.0    :  2.2.0 (OK)
sphinx >=0.6.6    :  1.8.2 (OK)
rope >=0.9.4      :  0.11.0 (OK)
jedi >=0.9.0      :  0.13.1 (OK)
psutil >=0.3      :  5.4.8 (OK)
nbconvert >=4.0   :  5.3.1 (OK)
pandas >=0.13.1   :  0.23.4 (OK)
numpy >=1.7       :  1.15.1 (OK)
sympy >=0.7.3     :  1.3 (OK)
cython >=0.21     :  0.29 (OK)
qtconsole >=4.2.0 :  4.4.2 (OK)
IPython >=4.0     :  7.1.1 (OK)
matplotlib >=2.0.0:  2.2.3 (OK)
pylint >=0.25     :  2.1.1 (OK)
```

**Comment by ccordoba12** [comment]
Thanks for reporting. This is fixed in our **3.3.2** version, which should be available in Anaconda in a couple of days (if you're using it), or you can use it with pip already.

**Comment by gzhechu** [comment]
I've upgraded to 3.3.2 via both Anacona and pip3. Thanks.
